src/content/devices/input-output/js/main.js

```diff
@@ -71,10 +65,10 @@ function attachSinkId(element, sinkId) {
       var errorMessage = error;
       if (error.name === 'SecurityError') {
         errorMessage = 'You need to use HTTPS for selecting audio output ' +
-            'device: ' + error;
+        'device: ' + error;
```

Comment:
Frustrating... Somehow I wound up reindenting these (and they got messed up in the process).